app.rb

```diff
@@ -0,0 +1,36 @@
+require_relative './book'
+require_relative './teacher'
+require_relative './student'
+require_relative './rental'
+require_relative './person'
+require 'json'
+require_relative './user_input'
+require_relative './user_output'
+
```

Comment:
- [ ] OPTOINAL
I suggest you add more details of a book or a person so as to make the app flow more intuitive for users. See the highlighted parts of the screenshot below for the same:

**Your interactive app:**

![image](https://user-images.githubusercontent.com/12745474/202794637-759bdec7-2ac3-44da-84a1-424d9f075600.png)


**Expected:**
![image](https://user-images.githubusercontent.com/12745474/202795065-47f1bc0a-f2bb-4439-b19b-addcea8a6239.png)

Reply:
Thanks, @rloterh  but I don't seem to understand what you ask me to do

Reply:
My code looks exactly like this, but I don't seem to get what you're asking for!

Reply:
My code work exactly as that too